integreat_cms/deepl_api/deepl_api_client.py

```diff
@@ -141,6 +141,17 @@ def translate_queryset(self, queryset, language_slug):
                     target_language.slug
                 )
 
+                # Before translating, check if translation is not up-to-date
+                if existing_target_translation.translation_state in (
+                    "UP_TO_DATE",
+                    "MACHINE_TRANSLATED",
+                ):
+                    messages.error(
+                        self.request,
+                        _("There already is an up-to-date translation"),
```

Comment:
```suggestion
                        _("There already is an up-to-date translation for {}").format(
                            content_object.best_translation.title
                        ),
```

It's more user friendly to indicate for which content a translation already actual and for which not especially when multiple of contents are translated at once.

Reply:
I had wanted to do that too, but couldn't get it to work so pylint wouldn't complain. But this seems to work. Thank you!